### ISSUE TITLE
thanks for your work, keep it up!

### ISSUE THREAD

**Issue by Andry042023** [chrskerr/task-list-kanban#16]
thanks for your work, keep it up!
ready to support with a donation

It would also be great to have such functions
1. Viewing archive cards
2. Ability to collapse columns
3. write subtasks
- [ ]
- [ ]
4. ability to organize tags #type/done

**Comment by Beingpax** [comment]
Yes, the ability to add and render subtasks would be a great addition. Hope it gets added.

**Comment by ErikaRS** [comment]
Thanks! I appreciate the feedback. I've split the feature requests into more targeted issues:

- [View archived cards](https://github.com/ErikaRS/task-list-kanban/issues/73)
- [Ability to collapse columns](https://github.com/ErikaRS/task-list-kanban/issues/74)
- [Support subtasks](https://github.com/ErikaRS/task-list-kanban/issues/75)
- [Ability to organize tags](https://github.com/ErikaRS/task-list-kanban/issues/76)

Ability to save tags in the filter was recently implemented and should be available now!
